contrib/test-venv.sh

```diff
@@ -2,5 +2,24 @@
 
 VENV=$(dirname $0)/..
 BUILD=${VENV}/build
+INSTALLED_TESTS=${VENV}/dist/share/installed-tests/fwupd
+export G_TEST_BUILDDIR=${INSTALLED_TESTS}
+export G_TEST_SRCDIR=${INSTALLED_TESTS}
+export LIBFWUPD_BUILD_DIR=${BUILD}/libfwupd
+export DAEMON_BUILDDIR=${BUILD}/src
+export PATH=${VENV}/bin:$PATH
 
+echo "Build time test suite"
 ninja -C ${BUILD} test
+
+echo "Testing fwupdtool.sh"
+${INSTALLED_TESTS}/fwupdtool.sh
+
+echo "Starting daemon"
+G_DEBUG=fatal-criticals ${VENV}/bin/fwupd --verbose --no-timestamp >fwupd.txt 2>&1 &
```

Comment:
I guess arguably this could be put in `venv/` somewhere, like `venv/dist`, but it is put here because fwupd.sh looks for it at the place it was launched, so other CI would need to be changed too.

LMK if you have strong opinions.

Reply:
no strong opinions either way.